CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ set(POCL_INSTALL_PUBLIC_LIBDIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
 set(POCL_INSTALL_PRIVATE_LIBDIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pocl" CACHE PATH "POCL private libdir")
 
 # for pocl.icd
-if(UNIX AND NOT CMAKE_CROSSCOMPILING AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+if(UNIX AND (NOT CMAKE_CROSSCOMPILING) AND (CMAKE_INSTALL_PREFIX STREQUAL "/usr"))
```

Comment:
the CMake default is /usr/local, like autotools. So how it worked before: with /usr/local, pocl.icd went to /etc, but with /usr it went to /usr/etc. That doesn't make sense IMO. I believe the standard prefix for system-wide install is /usr, so for that case we should install pocl.icd into /etc. For /usr/local i dunno, with this patch it ends up in /usr/local/etc.